packages/api-client/src/TamanuApi.js

```diff
@@ -118,7 +118,7 @@ export class TamanuApi {
   async fetch(endpoint, query = {}, moreConfig = {}) {
     const { headers, returnResponse = false, throwResponse = false, ...otherConfig } = moreConfig;
     const queryString = qs.stringify(query || {});
-    const path = `${endpoint}${query ? `?${queryString}` : ''}`;
+    const path = `${endpoint}${queryString ? `?${queryString}` : ''}`;
```

Comment:
Fix issue where a ? is added to every request url even if there is no query string